### PR TITLE
DW-7303: Add AP domain to CHROME_OPTS host-rules

### PR DESCRIPTION
**Pull request by nathanbigaignon**

### CODE REVIEW

File: src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt [hunk 1]
```diff
@@ -214,7 +214,8 @@ class TaskDeploymentService {
             ".${configurationResolver.getStringConfig(ConfigKey.AWS_REGION)}.compute.internal",
             ".dataworks.dwp.gov.uk",
             "localhost",
-            "127.0.0.1"
+            "127.0.0.1",
+            ".ap.dataworks.dwp.gov.uk"
```
Comment (IanUtley) [inline src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt:218]: Isn't this covered by .dataworks.dwp.gov.uk at line 215?

File: src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt [hunk 2]
```diff
@@ -353,7 +354,7 @@ class TaskDeploymentService {
                         "--disable-infobars",
                         "--disable-features=TranslateUI",
                         "--disk-cache-dir=/dev/null",
-                        "--host-rules=\"MAP * 127.0.0.1, MAP * localhost, EXCLUDE github.ucds.io, EXCLUDE git.ucd.gpn.gov.uk, EXCLUDE azkaban.workflow-manager.dataworks.dwp.gov.uk\"",
+                        "--host-rules=\"MAP * 127.0.0.1, MAP * localhost, EXCLUDE github.ucds.io, EXCLUDE git.ucd.gpn.gov.uk, EXCLUDE azkaban.workflow-manager.dataworks.dwp.gov.uk, EXCLUDE *.ap.dataworks.dwp.gov.uk\"",
```
Comment (IanUtley) [inline src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt:357]: As above.

File: src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt (hunk 1 above)
Comment (ChrisScottThomas) [inline src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt:218]: line 215 covers this.

Comment (nathanbigaignon) [reply src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt]: Yes  it is. Thanks, I am removing it.

Comment (nathanbigaignon) [reply src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt]: Thanks, removed now.

**Review by IanUtley — APPROVED** [review]

**Pull request merged**